Sources/Shared/Models/VIMPictureCollection.h

```diff
@@ -32,6 +32,7 @@
 
 @property (strong, nonatomic, nullable) NSString *uri;
 @property (strong, nonatomic, nullable) NSArray *pictures;
+@property (nonatomic, assign) BOOL isActive;
```

Comment:
Are value-type properties in Objective-C `assign` by default?

Reply:
I think so, but we've used `assign` explicitly elsewhere in this project so wanted to be consistent.